examples/device-dashboard/Makefile

```diff
@@ -20,27 +20,32 @@ ifeq ($(OS),Windows_NT)   # Windows settings. Assume MinGW compiler. To use VC:
   MAKE += WINDOWS=1 CC=$(CC)
 endif
 
-all: $(PROG)              # Default target. Build and run program
-	$(RUN) ./$(PROG) $(ARGS)
+# Default target. Build and run program
+all: $(PROG)
+	$(RUN) $(PROG) $(ARGS)
 
-# Before embedding files, gzip them to save space
-packed_fs.c: ca.pem $(FILES_TO_EMBED) Makefile
-	$(CC) ../../test/pack.c -o $(PACK)
-ifeq ($(OS),Windows_NT)
-	$(PACK) ca.pem $(FILES_TO_EMBED) > $@
-else
-	rm -rf tmp/web_root && mkdir tmp && cp -r web_root tmp/ && cp -f ca.pem tmp/
-	cd tmp && echo $(FILES_TO_EMBED) | xargs -n1 gzip && ../pack ca.pem `find web_root -type f` > ../$@
-endif
-
-$(PROG): $(SOURCES)       # Build program from sources
+# Build program from sources
+$(PROG): $(SOURCES)
 	$(CC) $(SOURCES) $(CFLAGS) $(CFLAGS_MONGOOSE) $(CFLAGS_EXTRA) $(OUT)
 
-clean:                    # Cleanup. Delete built program and all build artifacts
-	$(DELETE) $(PROG) *.o *.obj *.exe *.dSYM pack tmp mbedtls
+# Bundle JS libraries (preact, preact-router, ...) into a single file
+web_root/bundle.js:
+	curl -s https://npm.reversehttp.com/preact,preact/hooks,htm/preact,preact-router -o $@
 
-# see https://mongoose.ws/tutorials/tls/#how-to-build for TLS build options
+# Create optimised CSS. Prerequisite: npm -g i tailwindcss tailwindcss-font-inter
+web_root/main.css: web_root/index.html web_root/main.js
+	npx tailwindcss -o $@ --minify
```

Comment:
This requires `npm` being installed, not standard

Reply:
No more gzipping ?

Reply:
No more gzipping, that's too much IMO.
Yeah, npm needs to be installed - in the tutorial, we could just mention so.